cmd/api/api.go

```diff
@@ -8,21 +8,21 @@ import (
 	"time"
 
 	"github.com/gitpods/gitpods/handler"
-	"github.com/gitpods/gitpods/store"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/go-kit/kit/metrics/prometheus"
-	"github.com/gorilla/sessions"
 	"github.com/oklog/oklog/pkg/group"
 	prom "github.com/prometheus/client_golang/prometheus"
 	"github.com/urfave/cli"
 )
 
 const (
-	FlagAddr     = "addr"
-	FlagEnv      = "env"
-	FlagLogLevel = "loglevel"
-	FlagSecret   = "secret"
+	FlagAddr               = "addr"
```

Comment:
hmm. I can't find any documentation about Destination. Can you post a short example from somewhere it is in use? Thanks!